rest/revocation_test.go

```diff
@@ -1868,6 +1868,82 @@ func TestReplicatorRevocationsWithTombstoneResurrection(t *testing.T) {
 	assert.NoError(t, err)
 }
 
+func TestReplicatorRevocationsWithChannelFilter(t *testing.T) {
+	base.RequireNumTestBuckets(t, 2)
+
+	base.SetUpTestLogging(t, base.LevelDebug, base.KeyAll) // CBG-1981
+
+	// Passive
+	_, rt2 := InitScenario(t, nil)
+	defer rt2.Close()
+	rt2_collection := rt2.GetSingleTestDatabaseCollection()
+
+	// Active
+	rt1 := NewRestTesterDefaultCollection(t, //  CBG-2319: replicator currently requires default collection
+		&RestTesterConfig{
+			CustomTestBucket: base.GetTestBucket(t),
+		})
```

Comment:
This should pick up a new bucket by default, I think you can pass `nil` as the seoncd argument